packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -161,6 +161,14 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
       [TelemetryProperty.AccountType]: AccountType.Azure,
     });
     await vscode.commands.executeCommand("azure-account.login");
+    const azureAccount = this.getAzureAccount();
+    if (azureAccount.status != loggedIn) {
```

Comment:
Does CLI also have this bug?

Reply:
CLI Azure login is handled by codeFlow and will throw error if timeout. VsCode azure-account-extension will catch this error and cause this bug.